tests/integration_tests/test_integration_dds.py

```diff
@@ -253,14 +253,79 @@ def test_can_create_two_projects(self):
         self.assertNotEqual(dds_project_id1, dds_project_id2)
 
 
+class TestIntegrationDDSShortWait(BaseIntegration):
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.mock_duration = 2
+
+    @gen_test(timeout=2+1)
+    def test_mock_duration_is_10(self):
```

Comment:
Incorrect name? :thinking:

Reply:
Aha yes, I think I realized the test did not have to take that long and shortened it at some point but forgot to edit the name :sweat_smile: